library/core/src/ptr/mut_ptr.rs

```diff
@@ -1112,6 +1112,42 @@ impl<T> *mut [T] {
         // SAFETY: the caller ensures that `self` is dereferencable and `index` in-bounds.
         unsafe { index.get_unchecked_mut(self) }
     }
+
+    /// Returns the pointee as a unique, uninitialized slice.
+    ///
+    /// # Safety
+    ///
+    /// Behavior is undefined if any of the following conditions are violated:
+    ///
+    /// * The pointer must be [valid] for reads and writes for `ptr.len() * mem::size_of::<T>()` many bytes,
+    ///   and it must be properly aligned. This means in particular:
```

Comment:
```suggestion
    ///     and it must be properly aligned. This means in particular:
```

Reply:
I'm sorry, I don't understand what you mean. Are you talking about the term "properly aligned" or the indentation of the commentary? According to your edit, you meant the latter, but I don't get it.

Reply:
Ah lol "alignment" is ambiguous. ;)

I meant the indention. But I just realized you probably deliberately made these next two bullets further indented as sub-bullets. Let's hope this renders correctly, which I find really hard to predict with markdown.

Reply:
The comment is largely copied from `slice::from_raw_parts(_mut)`, so it should work :)